### ISSUE TITLE
[Request] ZombiU graphics pack

### ISSUE THREAD

**Issue by LuismaSP89** [slashiee/cemu_graphic_packs#272]
ZombiU works perfect in the latest release of cemu (1.14.0) so, I think it´s time to add a graphics pack for this awesome game, at least for the resolution.

**Comment by getdls** [comment]
I have a basic pack up and running...
But It'll need some more love  before committing.  Chromatic aberration and noise fixes among other things. 

<img width="1131" alt="zoombieu" src="https://user-images.githubusercontent.com/25286924/48674167-90d7c580-eb49-11e8-9b5e-da8f27743b87.png">

**Comment by getdls** [comment]
Initial commit done.  

Known issues: 

- Texture jitter when rotating view, Cemu issue
- Shadows wonkey, Cemu issue
- 5120x2880 some lights too bright, graphic pack issue.. Need Nsight to check-> later 

Anything else, report back and we'll see what we can do.

**Comment by getdls** [comment]
Fixed colour banding/truncation
<img width="1368" alt="moarcolour" src="https://user-images.githubusercontent.com/25286924/48678540-fac19080-eb84-11e8-8323-b6b8718c7a6f.png">

**Comment by LuismaSP89** [comment]
A new bug was introduced in 1.14.0 Graphic Packs: V210, in this case there´s a lot of flickering, you can see by moving quickly the camera from side to side like saying "no"

Another bug encountered is that the lights came across the walls, "all emergency lights, red and white" but this happens in the previous graphic pack too.

**Comment by getdls** [comment]
This kind of flickering, like the light is lagging behind?
I get that with graphics pack disabled. 

![flickering](https://user-images.githubusercontent.com/25286924/48679849-4ed47100-eb95-11e8-87bb-61d5a8558028.jpg)

**Comment by LuismaSP89** [comment]
This one it´s only 30fps recorded with my mobile phone (Ignore the all-blue color, I don´t know why the record looks all blue) but you can see a bit better the clipping I said:

[Record.zip](https://github.com/slashiee/cemu_graphic_packs/files/2593592/Record.zip)

**Comment by getdls** [comment]
Like this poster? Smearing  in the direction you are moving?   Also happens on the floor when looking up/down. 
![smearing](https://user-images.githubusercontent.com/25286924/48729558-2b163700-ec38-11e8-80d8-4c3bedf85e9a.jpg)

**Comment by getdls** [comment]
It happens for me without the graphics pack.. If its some sort of texture corruption, it does make sense that graphics pack can make more common. 

The biggest difference between the two versions  are format changes
```
[TextureRedefine]
width = 1280
height = 720
formats = 0x019,0x01a
overwriteFormat = 0x01f

[TextureRedefine]
width = 320
height = 192
formats = 0x01a,0x019
overwriteFormat = 0x01f

[TextureRedefine]
width = 320
height = 180
formats = 0x01a,0x019
overwriteFormat = 0x01f
```

Add # and see if un-commenting them helps. 
```
#[TextureRedefine]
#width = 1280
#height = 720
#formats = 0x019,0x01a
#overwriteFormat = 0x01f

```

**Comment by LuismaSP89** [comment]
Ok, I tested with your latest release "V.214" I don´t know what you changed but now the problem is solved.

**Comment by getdls** [comment]
:/   I added more stuff.. so it should have been even worse. 
I don't doubt that you, its just that this sounds more like luck if it breaks or works, not because of the graphics pack.  For example when I switch between full-screen and windowed it tends to be good for a while.

Anyway if you want to check out different versions and compare, all old builds are here.  
https://github.com/slashiee/cemu_graphic_packs/releases

**Comment by LuismaSP89** [comment]
Edit: Deleted my last comment, because I was testing with the resolution pack disabled, so forget it, It solves nothing, sorry :S

**Comment by LuismaSP89** [comment]
Ok, after some more and more testing, I found the problem.

The "lights came across the walls" is a Cemu´s problem, indeed, it happens too with graphic pack disabled.

All the versions of the graphic pack "v.208,210, 214" has the flickering problem for me, I thought that the first pack is Ok, because, sometimes, when I delete the folder "ZombieU" containing the graphic pack and copy the new one, cemu disables it, (I think this doesn´t happen with other graphic packs, since the lesslentdirt doesn´t have this problem) and I was running the game with the graphics pack disabled.

So, in short:

Light across the walls - Cemu problem
All resolution graphic packs have the flickering problem, this doesn´t happen with the resolution graphic pack disabled (at least for me) maybe (If this happens for you even withouth graphic pack) you have the vsync disabled? I have enabled for all apps in my Nvidia control panel. But this doesn´t explains why the problem happens with the resolution pack enabled.

**Comment by LuismaSP89** [comment]
Tested 5 or 6 times and now without the graphic pack has the flickering too, I don´t know why this happens now since I tested a lot of times before and worked Ok with the graphic pack disabled... this game are trolling me... ¬¬ sorry, closing the issue, let´s hope that the Cemu team solves the remaining issues, thanks a lot for your help and my apologies for wasting your time.
